rbd/errors.go

```diff
@@ -71,6 +71,12 @@ var (
 	// revive:enable:exported
 )
 
+// Public general error
+const (
+	// ErrNotExist indicates a non-specific missing resource.
+	ErrNotExist = rbdError(-C.ENOENT)
```

Comment:
Ah, I thought you would make that a simple string error. But that works too. I hope the String() function of the rbdError will not add some text like "file or directory not found"?

Reply:
It does but what matters is the errno. Plus we do this same thing in rados already and we can tweak the text in the future since this error type is not defined by the text.